includes/Core/Authentication/Authentication.php

```diff
@@ -1151,4 +1201,30 @@ private function get_proxy_permissions_url() {
 			admin_url( 'index.php' )
 		);
 	}
+
+
+	/**
+	 * Verifies the user input settings
+	 *
+	 * @since n.e.x.t
+	 */
+	private function verify_user_input_settings() {
+		if (
+			User_Input_State::VALUE_COMPLETED === $this->user_input_state->get()
```

Comment:
This was the requirement per the (original) ACs, however during the IB discussion it was decided to introduce `VALUE_MISSING` which means that the only time this check should be run is if the setting value is currently empty, i.e. this clause should return if the state value is anything that's not empty.